src/data/instant-observability-categories.json

```diff
@@ -90,5 +90,12 @@
     "associatedKeywords": [
       "mlops"
     ]
+  },
+  {
+    "value": "notifications",
+    "displayName": "Notifications",
+    "associatedKeywords": [
+      "notifications",
```

Comment:
```suggestion
      "notifications"
```

Reply:
I think the trailing comma is what is causing it to fail since JSON doesn't support trailing commas

Reply:
That's the culprit.. 🥇